src/main/java/com/ipiecoles/java/eval/mdd050/service/AlbumService.java

```diff
@@ -3,17 +3,25 @@
 import com.ipiecoles.java.eval.mdd050.model.Album;
 import com.ipiecoles.java.eval.mdd050.repository.AlbumRepository;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataIntegrityViolationException;
 import org.springframework.stereotype.Service;
 
 @Service
 public class AlbumService {
-
+	
     @Autowired
     private AlbumRepository albumRepository;
 
-
-    public Album creerAlbum(Album album) {
+    
+    
+    // Gestion d'erreur lazy, je sais !
+    
+    public Album creerAlbum(Album album) throws DataIntegrityViolationException{
```

Comment:
La `DataIntegrityViolationException` est une `RuntimeException`, pas besoin de la propager explicitement
```suggestion
    public Album creerAlbum(Album album) {
```